README.md

```diff
@@ -33,6 +33,13 @@ Configure inside your `webpack.config.js`:
         test: /\.(html|svelte)$/,
         exclude: /node_modules/,
         use: 'svelte-loader'
+      },
+      {
+        // required to prevent errors from Svelte on Webpack 5+
+        test: /svelte\/.*\.mjs$/,
```

Comment:
This should be limited to just the `node_modules` directory to prevent changing how files are resolved in user code. (this could cause confusion if users had a `svelte` directory and tried to resolve `.mjs` files in that directory)
```suggestion
        test: /node_modules\/svelte\/.*\.mjs$/,
```

Reply:
Wow, didn't realize I could just accept your suggestion right here. :confused: 
This still could be handy later.
Thanks.